tests/query_tests.rs

```diff
@@ -63,56 +63,6 @@ fn query() {
     });
 }
 
-#[test]
```

Comment:
Why is this test deleted?

Reply:
It was in fact testing a bug on Couchbase side. The assert passsed with a wrong result.

By updating the version of libcblite, the test failed because the bug was resolved.

Yep 🤯